app/assets/stylesheets/custom.css

```diff
@@ -0,0 +1,3 @@
+#nolink { 
+    text-decoration:none; 
+}
```

Comment:
Why a seperate file for this?

Reply:
Just wanted to make sure that all the required css lies in one place. Is there any drawback to this sir?

Reply:
You should break your css into files section wise. For eg: One css to show your TODOs and another file could be to edit them. In this way, you will be able to reuse most of your CSS as well